crawler/urlJobProcessorProbes/sw.js

```diff
@@ -0,0 +1,17 @@
+const swRegexp = /\.serviceWorker\.register/;
```

Comment:
Matching for `serviceWorker` and `register` separately as will help to detect `const sw = navigator.serviceWorker; sw.register …`